edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+  * There is definitive proof of the accused person committing theft (such as a witness seeing them steal, CCTV footage, admission of guilt, etc.).
+  * The suspect has been contacted and asked to return stolen items.
+  * The suspect has been unresponsive/uncooperative and has not returned stolen items.
+* If you receive reports that lead you to believe that there has been theft, then do your best to resolve the situation during the competition if possible.
+  * Make an announcement, giving a description of stolen items and advising competitors to give items to an organizer or Delegate if they are found. In past incidents, competitors have returned stolen items after similar announcements.
+  * Ask competitors if they witnessed anything that could be related to the theft (e.g. someone standing next to a table where puzzles were stolen or seeing someone playing with a puzzle that went missing).
+  * Talk to any suspects and ask direct questions. If they confess and return any stolen items, thank them for their honesty and warn them to avoid stealing in the future. Keep the age and maturity of the individual in mind.
+  * Ask the venue for security camera footage to identify potential suspects if possible.
+* If puzzles were not returned at the competition, but you believe you know who stole them then contact them and ask for the puzzles to be returned. Contact the WDC if the individual is not responsive or is unwilling to return the puzzles.
+* Report incidents of theft to the WDC so that we can note down competitors names in case there are repeated cases of theft in the future.
+
+# Handling Behavioral Issues of Spectators at Competitions {.page-break-before}
+
+* Local Delegates should do their best to handle any issues that arise with spectators in a respectful manner, remembering that they are acting as official representatives of the WCA.
+* Be aware of venue rules and policies. You may be able to ask an official from the venue or local law enforcement to remove individuals causing problems if needed. 
+* Contact local law enforcement immediately if there is extreme behavior that may threaten competitor safety. Once the immediate concerns of safety have been addressed, contact the WDC to report the details of what happened.
+
+# Online Interactions (WCA Forum/Official WCA Platforms)
+
+* If competitors/parents/others are being impolite or inappropriate in their online interactions related to competitions, then please do your best to respectfully handle the situation (please keep in mind that you are a representative of the WCA). You may reach out to the WDC for advice on how to best handle the situation if the problem persists or you are unsure of how to proceed.
```

Comment:
I think it is worth mentioning that the WDC's jurisdiction specifically does not extend outside of WCA-controlled environments before this point is made, as it is not clear up to this point when reading the document for the first time.